sonar-db/src/main/java/org/sonar/db/organization/DefaultTemplates.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.db.organization;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nullable;
+
+import static java.util.Objects.requireNonNull;
+
+public class DefaultTemplates {
+  private String project;
+  private String view;
+
+  public String getProject() {
+    checkProjectNotNull(this.project);
+    return this.project;
+  }
+
+  public DefaultTemplates setProject(String project) {
```

Comment:
Another option of the upper problem is to accept the parameter `PermissionTemplateDto` instead of a String. It's possible according because all callers do `permissionTemplateDto.getUuid()`.